bootstrap.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 set -eu
 
-KUBECTL_VERSION=v1.24.16
-HELM_VERSION=v3.12.2
+KUBECTL_VERSION=v1.29.2
+HELM_VERSION=v3.14.2
```

Comment:
These can be bumped to the latest available versions now that the cluster upgrade is complete.